features/support/hooks.rb

```diff
@@ -1,4 +1,5 @@
-require 'color'
+BOLD  = "\033[1m"
```

Comment:
We should just use the colorize gem ....

Reply:
installing one gem just for this case? isn't this too much? 🤔

Reply:
i don't know that gem, but if u end up installing it, do it only for the test environment

Reply:
it is already installed `bundle show colorize`